src/index.html

```diff
@@ -4,9 +4,383 @@
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a
+              href="#computers"
+              class="nav__link"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tabs" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="main-container">
+      <main class="main">
+        <section class="prd-card main__prd-card-container" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="apple a1419 imac 27 retina 5K"
+            class="prd-card__image"
+          >
+          <h1 class="prd-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <p class="prd-card__code">
+            Код товара: 195434
+          </p>
+          <div class="prd-card__rating">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="prd-card__reviews">
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="prd-card__price">
+            <p class="prd-card__price-label">
+              Цена
+            </p>
+            <p class="prd-card__price-number">
+              69 999 грн
+            </p>
+          </div>
+          <a
+            href="#"
+            data-qa="hover"
+            class="prd-card__buy"
+          >
+            купить
+          </a>
+        </section>
+        <section class="prd-card main__prd-card-container" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="apple a1419 imac 27 retina 5K"
+            class="prd-card__image"
+          >
+          <h1 class="prd-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <p class="prd-card__code">
+            Код товара: 195434
+          </p>
+          <div class="prd-card__rating">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
```

Comment:
Stars should work correctly with `stars--4` and other modifiers, like in previous task.

Reply:
I've just made it as I have it my previous task